spec/README.md

```diff
@@ -0,0 +1,66 @@
+# Atomistics
+The `atomistics` package provides "interfaces for  atomistic simulation codes and workflows". While the 
+[user documentation](https://atomistics.readthedocs.io/) already covers how to use the `atomistics` package, this 
+document explains the motivation and vision of the development the `atomistics` package, the underlying concepts and 
+the resulting internal structure of the `atomistics` package. 
+
+## Vision 
+The `atomistics` package provides "interfaces for  atomistic simulation codes and workflows".
```

Comment:
This part fails to define what `atomistics` actually means. I think the easiest would be to base it on ASE Atoms. Then it would be something like "Atomistic simulation codes are those, which extract physical information of atomistic structures, given either by ASE Atoms or a set of properties that can be translated to ASE Atoms".

Reply:
It doesn't have to be this wording, but I think we need something that clearly defines the borders